mhkit/loads/extreme/mler_simulation.m

```diff
@@ -0,0 +1,54 @@
+function sim = mler_simulation(sim)
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+%     Define the simulation parameters that are used in various MLER
+%     functionalities. If no input is given, then default values are returned.  
+%     
+%     Parameters
+%     ----------
+%         sim : struct (optional)
+%             Simulation parameters
+%             Keys:
+%             -----
+%             'startTime': starting time [s]
+%             'endTime': ending time [s]
+%             'dT': time-step size [s]
+%             'T0': time of maximum event [s]
+%             'startx': start of simulation space [m]
+%             'endX': end of simulation space [m]
+%             'dX': horizontal spacing [m]
+%             'X': position of maximum event [m]
+%         
+%      Returns
+%      -------
+%         sim : struct 
+%             Simulation parameters including spatial and time calculated
+%             arrays
+%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+if nargin < 1
+    sim.startTime = -150.0;  % .s Starting time
+    sim.endTime = 150.0;  % .s Ending time
+    sim.dT = 1.0;  % .s Time-step size
+    sim.T0 = 0.0;  % .s Time of maximum event
+
+    sim.startX = -300.0;  % .m Start of simulation space
+    sim.endX = 300.0;  % .m End of simulation space
+    sim.dX = 1.0;  % .m Horiontal spacing
+    sim.X0 = 0.0;  % .m Position of maximum event
+end
```

Comment:
Do we want to make sure the input is correct for this? We certainly don't have to since it will fail at lines 45-49 (except for T0 and X0 which aren't used). But if we do ...
else
       assert nargin(1) is struct and has fields {1 - 8}